science/pymol/Portfile

```diff
@@ -23,46 +22,34 @@ long_description    PyMOL is a molecular graphics system with an embedded Python
 
 homepage            https://www.pymol.org/
 
-checksums           rmd160  e7a962473732172b370e6c7a880384e7a1ffa62e \
-                    sha256  b0af15082c44c92c285cab341506da50407619f5dee09d98fdf2802e356219fd \
-                    size    10552074
+checksums           rmd160  6432d47616aee1e5d9d40cc2a74425b33159d7a0 \
+                    sha256  67c5aa944ce9ac2e3babd42e7e2b78ed11719283a2c0a063e100cfe24665c0d3 \
+                    size    10471287
 
 subport pymol-devel {
     github.setup    schrodinger pymol-open-source 4a3c72ace1f8e39e80db4026d45222f8b7a44169
     version         20210428-[string range ${github.version} 0 7]
     conflicts       pymol
     revision        0
     maintainers     {reneeotten @reneeotten} openmaintainer
-    checksums       rmd160  b8f596935a043a3ecde7b8f0e1c914b74277d1d0 \
-                    sha256  92f39021487fa1c3da1840721c7ad4c3992dfcb54d73fef27fb30842828425a8 \
-                    size    10469954
+    checksums       rmd160  6432d47616aee1e5d9d40cc2a74425b33159d7a0 \
+                    sha256  67c5aa944ce9ac2e3babd42e7e2b78ed11719283a2c0a063e100cfe24665c0d3 \
+                    size    10471287
 }
 
-variant python27 conflicts python36 python37 python38 python39 description {Use Python 2.7} {}
 variant python36 conflicts python27 python37 python38 python39 description {Use Python 3.6} {}
 variant python37 conflicts python27 python36 python38 python39 description {Use Python 3.7} {}
 variant python38 conflicts python27 python36 python37 python39 description {Use Python 3.8} {}
 variant python39 conflicts python27 python36 python37 python38 description {Use Python 3.9} {}
 
-if {[variant_isset python27]} {
-    python.default_version 27
-} elseif {[variant_isset python36]} {
+if {[variant_isset python36]} {
     python.default_version 36
 } elseif {[variant_isset python37]} {
     python.default_version 37
 } elseif {[variant_isset python38]} {
     python.default_version 38
 } elseif {[variant_isset python39]} {
     python.default_version 39
```

Comment:
right now there is no default variant set anymore. The last two lines should be changed to:
```
} else {
    default_variants +python39
     python.default_version 39
```